hydroflow/src/builder/surface/pull_handoff.rs

```diff
@@ -20,6 +20,15 @@ where
         Self { port }
     }
 }
+impl<Hof> TrackPullDependencies for HandoffPullSurface<Hof>
+where
+    Hof: Handoff,
+{
+    fn insert_dep(&self, e: &mut super::DirectedEdgeSet) -> u16 {
+        let my_id = e.add_node(format!("Handoff_{}", self.port.handoff_id));
```

Comment:
nit: this can be inlined

Reply:
cleaned up as suggested by Mingwei: handoff ids now stored separately in the `DataflowGraphStorage` struct

Reply:
this got longer anyhow due to tracking handoff names separately